js/page-specific/animal-services/adopted-pets.js

```diff
@@ -1,69 +1,68 @@
 (function ($, BcList) {
-  /**
+	/**
    * MODIFY these values based on your configuration
    */
-  var listClass = 'adoptable-pet-list';
-  var tabSelector = '.adoptable-pet-tabs';
-  var dataUrl = "//egov.baltimorecountymd.gov/LostAdoptPetService/GetAdoptablePets";
-  var templateUrl = "/sebin/l/l/generic-pet.template.js?version=20170728";
+	var listClass = 'adoptable-pet-list';
+	var tabSelector = '.adoptable-pet-tabs';
+	var dataUrl = '//bcgis.baltimorecountymd.gov/LostAdoptPetService/GetAdoptablePets';
+	var templateUrl = '/sebin/l/l/generic-pet.template.js?version=20170728';
 
-  /**
+	/**
    * !!!! DO NOT MODIFY BELOW THESE LINES !!!!
    */
 
-  //Initialize Bc List
-  var adoptedPets = new BcList({
-    containerClass: listClass, //Selector of class you want to append your results
-    source: dataUrl,
-    templatePath: templateUrl
-  });
+	// Initialize Bc List
+	var adoptedPets = new BcList({
+		containerClass: listClass, // Selector of class you want to append your results
+		source: dataUrl,
+		templatePath: templateUrl
+	});
 
-  //Utils
-  function getURLParameter(name) {
-    return decodeURI(
-      (RegExp(name + '=' + '(.+?)(&|$)').exec(location.search) || [, null])[1]
-    );
-  }
+	// Utils
+	function getURLParameter(name) {
+		return decodeURI(
+			(RegExp(name + '=' + '(.+?)(&|$)').exec(location.search) || [, null])[1]
```

Comment:
Might I suggest you use the querystringer utility in /js/utilities?

Reply:
I am going to leave this for now, and if we choose to refactor this, I will pull in querystring and make it so there isn't so much repeated code in each of these files.